pkg/kubelet/dockertools/manager.go

```diff
@@ -216,12 +218,30 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 	}
 	dm.runner = lifecycle.NewHandlerRunner(httpClient, dm, dm)
-	if serializeImagePulls {
-		dm.imagePuller = kubecontainer.NewSerializedImagePuller(kubecontainer.FilterEventRecorder(recorder), dm, imageBackOff)
+	dm.containerGC = NewContainerGC(client, containerLogsDir)
+
+	minimumRequiredDockerAPIVersionForParallelImagePulls := false
```

Comment:
NIT: this variable name is quite long -- you may want to consider a slightly shorter name.